src/Components/Nav/Nav.css

```diff
@@ -0,0 +1,36 @@
+.nav-container {
```

Comment:
Thanks for starting the beginning sepcts of styling and getting the Nav bar together! Great Start!